app/presenters/iiif_print/iiif_manifest_presenter_behavior.rb

```diff
@@ -3,6 +3,12 @@ module IiifPrint
   module IiifManifestPresenterBehavior
     extend ActiveSupport::Concern
 
+    def manifest_metadata
+      current_ability = try(:ability) || try(:current_ability)
+      base_url = try(:base_url) || try(:request)&.base_url
+      IiifPrint.manifest_metadata_for(work: model, current_ability: current_ability, base_url: base_url)
```

Comment:
Do you want to memoize this?

Reply:
hmm i'm thinking no since this is for the overall manifest, later on it happens for each child work but i can do a test

Reply:
great call! it does get called 4x from the IIIF Manifest gem